app/components/candidate_interface/continuous_applications/application_review_component.html.erb

```diff
@@ -1,5 +1,33 @@
 <%= render SummaryListComponent.new(rows: rows) %>
 
+<% if show_what_happens_next? %>
+  <div class="govuk-!-margin-bottom-6">
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">What happens next</h2>
+    <% case application_choice.status %>
+      <% when *%w(awaiting_provider_decision) %>
+        <p class="govuk-body">The provider will review your application and let your know when they have a made a decision. In the meantime, you can:</p>
+        <ul class="govuk-list govuk-list--bullet govuk-!-margin-left-8">
+          <li>contact the provider directly if you have any questions</li>
+          <li>find out more about <%= govuk_link_to('funding your training', t('get_into_teaching.url_funding_and_support'), target: '_blank') %></li>
+          <li>get help from a <%= govuk_link_to('teacher training advisor', t('get_into_teaching.url_get_an_adviser_signup'), target: '_blank') %></li>
+        </ul>
+      </div>
+      <% when *%w(interviewing) %>
+        <p class="govuk-body">Congratulations on being invited for an interview! This is an important stage in successfully getting a place on a teacher training course.</p>
+        <p class="govuk-body">Learn more about what to expect and <%= govuk_link_to 'how to prepare for an interview', t('get_into_teaching.url_teacher_training_interview'), target: '_blank' %>.</p>
+      <% when *%w(inactive) %>
```

Comment:
Should we hard code application choice states in the `when` in the view? @JR-G / @vassyz what do you think about it?

Reply:
Not sure, I find `if application_choice.awaiting_provider_decision?` easier to read.

Haven't put much thought into it, but I'd probably implement this as

```ruby
def what_happens_next_view
  return unless ApplicationStateChange::DECISION_PENDING_AND_INACTIVE_STATUSES.include?(@application_choice.status.to_sym)

  @application_choice.status
end
```

have a list of views (or components)

```
/views/what_happens_next/awaiting_provider_decision.html.erb
/views/what_happens_next/interviewing.html.erb
/views/what_happens_next/inactive.html.erb
```

and in `app/components/candidate_interface/continuous_applications/application_review_component.html.erb` render `what_happens_next_view`.

Reply:
I'm very happy to see you suggest this @vassyz.
I've been wanting to raise a discussion around this format/style for a while. Maybe we can have a discussion about this soon.

Reply:
I've spent some time investigating the use of partials with ViewComponents and with using partials in the sidecar. None of the options are great. We can have a chat about it if we want.

I've just made the recommended changes.